exonum-java-binding-common/src/main/java/com/exonum/binding/common/message/TransactionMessage.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.common.message;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.exonum.binding.common.crypto.CryptoFunction;
+import com.exonum.binding.common.crypto.KeyPair;
+import com.exonum.binding.common.crypto.PublicKey;
+import com.exonum.binding.common.hash.HashCode;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+/**
+ * An Exonum transaction message.
+ */
+public interface TransactionMessage {
+  int AUTHOR_PUBLIC_KEY_OFFSET = 0;
+  int CLS_OFFSET = 32;
+  int TAG_OFFSET = 33;
+  int SERVICE_ID_OFFSET = 34;
+  int TRANSACTION_ID_OFFSET = 36;
+  int PAYLOAD_OFFSET = 38;
+
+  int AUTHOR_PUBLIC_KEY_SIZE = 32;
+  int SIGNATURE_SIZE = 64;
+
+  /**
+   * Returns a public key of the Transaction Message's author.
```

Comment:
Here and elsewhere, I'd suggest to not capitalize the individual words that comprise the interface name when they refer not to the interface, but to the thing it represents (a transaction message).

I.e., "Returns a public key of the author of the transaction message".

Also, if I remember correctly, `’s` is used with people or organizations to specify possession.